### PR TITLE
Bug 1619645 -- Improve wording for Magnitude of Difference aria-label

### PR DESCRIPTION
**Pull request by Tintinator**

### CODE REVIEW

**Comment by sarah-clements** [comment]
@Tintinator Is this ready for review?

**Comment by Tintinator** [comment]
Yep!

**Comment by sarah-clements** [comment]
> Yep!

Ok, I'll review it today. Next time you have a pr ready for review, please at-mention me or camd.

File: yarn.lock [hunk 1]
```diff
@@ -2367,6 +2367,11 @@ bn.js@^5.1.1:
   resolved "https://registry.yarnpkg.com/bn.js/-/bn.js-5.1.3.tgz#beca005408f642ebebea80b042b4d18d2ac0ee6b"
   integrity sha512-GkTiFpjFtUzU9CbMeJ5iazkCzGL3jrhzerzZIuqLABjbwRaFt33I9tUdSNryIptM+RxDet6OKm2WnLXzW51KsQ==
 
+bn.js@^5.1.1:
```
Comment (sarah-clements) [inline yarn.lock:2370]: I think you accidentally added this. You shouldn't be adding any yarn file updates unless you've added a package to package.json. The best way to keep yarn up to date is to pull directly from upstream, not your fork, and always run `yarn` before `yarn start:stage` for local development.

**Review by sarah-clements — APPROVED** [review]
Looks good - thanks :)

**Pull request merged**